whitelist.txt

```diff
@@ -52,6 +52,10 @@ dest
 fromhex
 preimage
 substring
+astuple
```

Comment:
```suggestion
```
i don't see this used anywhere?

Reply:
[Part of #49] I will remove this. Thanks.